src/reduced_basis/rb_construction.C

```diff
@@ -1453,6 +1453,20 @@ void RBConstruction::train_reduced_basis_with_POD()
     }
   libMesh::out << std::endl;
 
+  if (normalize_solution_snapshots)
+  {
+    std::cout << "Normalizing solution snapshots" << std::endl;
```

Comment:
We use `libMesh::out` for print statements inside the library.

Reply:
oops, thanks, fixed in the update I just pushed